### PR TITLE
Notes from Nov 15th meeting

### PR DESCRIPTION
**Pull request by lmbr-pip**
Signed-off-by: Pip Potter <61438964+lmbr-pip@users.noreply.github.com>

### CODE REVIEW

File: meetings/notes/sig-meeting-20221115.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Meeting Details
+
+- **Date/Time:** [November 15th, 2022 @ XX:XXpm UTC / XX:XXpm ET](https://lists.o3de.org/g/o3de-calendar/viewevent?repeatid=39350&eventid=1557573&calstart=2022-11-15)
```
Comment (allisaurus) [inline meetings/notes/sig-meeting-20221115.md:3]: need to update `XX:XXpm` ?

File: meetings/notes/sig-meeting-20221115.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+[Agenda Issue](https://github.com/o3de/sig-network/issues/79)
+
+## SIG Updates
+
+### General SIG announcements
+AWS is looking to contribute a [major update](https://github.com/o3de/o3de/issues/13198) to the 3rd person Multiplayer Sample
+
+[O3DCon 2022](https://events.linuxfoundation.org/o3dcon/) talks now on [YouTube](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ). Of interest to SIG:
+    * [Building Multiplayer Content for Large Worlds](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ)
+    * [Scale Testing Multiplayer Games on AWS](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ)
```
Comment (allisaurus) [inline meetings/notes/sig-meeting-20221115.md:17]: all the talk links in this list appear to be the playlist, not the individual talk links

Comment (lmbr-pip) [reply meetings/notes/sig-meeting-20221115.md:17]: Good catch, I started each talk and just copied the URL, I guess Youtube was tracking this via the playlist, will update.

**Review by AMZN-puvvadar — APPROVED** [review]
Looks good pending resolution of the issues @allisaurus has pointed out.

File: meetings/notes/sig-meeting-20221115.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## SIG Updates
+
+### General SIG announcements
+AWS is looking to contribute a [major update](https://github.com/o3de/o3de/issues/13198) to the 3rd person Multiplayer Sample
+
+[O3DCon 2022](https://events.linuxfoundation.org/o3dcon/) talks now on [YouTube](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ). Of interest to SIG:
+* [Building Multiplayer Content for Large Worlds](https://www.youtube.com/watch?v=mY8SgHPDVHA&list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=7)
+* [Scale Testing Multiplayer Games on AWS](https://www.youtube.com/watch?v=QayMbJxOzJ0&list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=17)
+* [Keynote: Supercharging Gameworld Performance Using the Cloud](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=20)
```
Comment (allisaurus) [inline meetings/notes/sig-meeting-20221115.md:18]: this link still resolves to the playlist

File: meetings/notes/sig-meeting-20221115.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+* [Keynote: O3DE on AWS The Future of 3D in the Cloud ](https://www.youtube.com/watch?v=jg_BJVNc5Xc&list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=26)
+* [Workshop: Building a Networked Game from Scratch: Multiplayer Drag Racing](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ&index=29)
```
Comment (allisaurus) [inline meetings/notes/sig-meeting-20221115.md:20]: same with this one (resolves to playlist)

File: meetings/notes/sig-meeting-20221115.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+   * lmbr-pip to look for volunteer external to SIG to help run SIG election, aim to hold elections first week in December so new chair(s) can be in place for December meeting.
+      * Election guide for all SIGs: https://github.com/o3de/community/blob/main/O3DE%20Elections/O3DE%20Elections%20Guide.md
+   
+## Open Discussion Items
+* None
+
+## Action Items
+Carried from [previous meeting](https://github.com/o3de/sig-network/blob/main/meetings/notes/sig-meeting-20220719.md):
+* _TODO_: Need to decide on SIG/network's responsibility towards cloud service gems once they migrate from O3DE/O3DE (may require charter update).
+
```
Comment (allisaurus) [inline meetings/notes/sig-meeting-20221115.md:56]: extra white space? (ignore if intended)

File: meetings/notes/sig-meeting-20221115.md (hunk 3 above)
Comment (lmbr-pip) [reply meetings/notes/sig-meeting-20221115.md:18]: All of the links to specific talks have `&index=x` which resolves to items in the play test. Theres a link to the playlist at the top on purpose.

Lines 16-18 link into the playlist as expected

File: meetings/notes/sig-meeting-20221115.md (hunk 4 above)
Comment (lmbr-pip) [reply meetings/notes/sig-meeting-20221115.md:20]: Again have that `index=x` which for me resolves directly to the talk

File: meetings/notes/sig-meeting-20221115.md (hunk 5 above)
Comment (lmbr-pip) [reply meetings/notes/sig-meeting-20221115.md:56]: Yes, intent is to separate carried from new items

File: meetings/notes/sig-meeting-20221115.md (hunk 3 above)
Comment (allisaurus) [reply meetings/notes/sig-meeting-20221115.md:18]: Hmm pasting that link into the browser just resolves to the top level playlist for me (https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ), in both Chrome and Firefox.

Line 17 actually does go directly to the Keynote, but with the playlist available in the right context menu, which seems like the desired behavior. The L17 and L18 links are formatted slightly differently but both contain the playlist & index at the end - maybe you intended to use the same format as L17, with the `watch` arg? : 
image

**Review by allisaurus — APPROVED** [review]

**Pull request merged**